### ISSUE TITLE
dotnet 2.1 required to run the dotnet-aspnetcodegenerator 2.2 tool

### ISSUE THREAD

**Issue by tuath** [aspnet/Docs#10016]
dotnet 2.1 required to run the dotnet-aspnetcodegenerator 2.2 tool.

Error message : 
The application to execute does not exist: 'C:\users\erik\.dotnet\tools\.store\dotnet-aspnet-codegenerator\2.2.0-rtm-35687\dotnet-aspnet-codegenerator\2.2.0-rtm-35687\tools\netcoreapp2.1\any\dotnet-aspnet-codegenerator.dll'.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 804e24dd-7e30-8cf5-2c99-1ffacb4582f6
* Version Independent ID: 3e25547b-e2c9-6634-2c38-2a77207f17d4
* Content: [Scaffold Identity in ASP.NET Core projects](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-2.2&tabs=netcore-cli)
* Content Source: [aspnetcore/security/authentication/scaffold-identity.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/security/authentication/scaffold-identity.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Hello @tuath ... It's a known issue. The fix is in I believe for the next patch release. See: https://github.com/dotnet/cli/issues/10459

**Comment by 7aylor** [comment]
If it helps you, I was able to get this working just by renaming some of the folders in the file path. If you browse to C:\users\erik.dotnet\tools.store\dotnet-aspnet-codegenerator\ on your PC, you'll probably see a folder called 2.2.0. I renamed that to 2.2.0-rtm-35687. There is another folder two folders deeper called 2.2.0, and I renamed that to 2.2.0-rtm-35687 as well. Once I did that, I was able to use the tool to scaffold without error. Hope this helps as a temporary work around.
